Source/EasyNetQ.Serialization.SystemTextJson/EasyNetQ.Serialization.SystemTextJson.csproj

```diff
@@ -11,4 +10,13 @@
     <ItemGroup Condition="'$(TargetFramework)'== 'netstandard2.0'">
       <PackageReference Include="System.Text.Json" />
     </ItemGroup>
+        <ItemGroup>
+      <PackageVersion Update="MinVer" Version="6.0.0" />
+    </ItemGroup>
+    <ItemGroup>
+      <PackageReference Update="MinVer">
+        <PrivateAssets>all</PrivateAssets>
+        <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
+      </PackageReference>
```

Comment:
Maybe this could be configured in the `Directory.Packages.props` with `GlobalPackageReference`